pkg/depository/get.go

```diff
@@ -66,6 +67,40 @@ func NewGetDepositoryCmd(option common.Options) *cobra.Command {
 			if host == "" {
 				return fmt.Errorf("no host provided")
 			}
+			certificate, _ := cmd.Flags().GetBool("certificate")
```

Comment:
add a new flag `certificate-language` which supports :
- "CN" 
- "ENG"

And this argument should be passed into http request with "?language="xxx""

Reply:
is it `?style=xxx` or `?language=ENG`?
the `bc-saas` displays `language := ctx.Query("language")`

Reply:
use `language` for now.